RevenueCatUI/Helpers/ProcessedLocalizedConfiguration.swift

```diff
@@ -0,0 +1,30 @@
+import RevenueCat
+
+/// A `PaywallData.LocalizedConfiguration` with processed variables
+struct ProcessedLocalizedConfiguration: PaywallLocalizedConfiguration {
```

Comment:
This is a "copy" of `PaywallData.LocalizedConfiguration` that allows us to make sure at compile time that we don't have any unprocessed variables.